src/app/utils/utils.js

```diff
@@ -195,89 +197,92 @@ const getFieldParam = (field) => {
 
 function collapse(results) {
   const owiLookup = {};
-  if (!results || !results.searchResults || !results.searchResults.itemListElement) return results
+  if (!results || !results.searchResults || !results.searchResults.itemListElement) return results;
```

Comment:
Oh I didn't know it is old. I just thought since they were really just missing `;` so I added them. I can of course reverse it back if it is better this way.